content/agent/kubernetes/daemonset_setup.md

```diff
@@ -140,6 +159,9 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: status.hostIP
+          # Trace Collection (APM) - comment this section to disable APM
```

Comment:
I'm repeating myself here, but it makes it really confusing to the user imo. We have one section here that they must `comment to disable APM`, and one line above that they should `uncomment to enable APM`. So what's the state by default? Hard to say. It's actually "enabled, harmless, but not actually usable` :/

Reply:
I'm tracking this feedback on our board, we will address it in another PR

Reply:
The process is not expensive if it does nothing (it's stuck listening to a channel so the Go runtime will optimize that) so as we discussed in our RFC, it's not a real issue. Users should enable APM easily so the "uncomment" is good. Maybe we can just remove this comment and that's all.

The end result is: "the Trace Agent doesn't collect traces, if you uncomment we'll start receiving your traces"